### PR TITLE
remove git link from requirements.txt

### PR DESCRIPTION
**Pull request by nnrepos**
first half of #550

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
 selenium==4.5.0
 six==1.16.0
 taskcluster==44.22.1
-firefox_code_coverage@https://github.com/marco-c/firefox-code-coverage/archive/36d00cfd1e8dd29649dc9696e97a0f28d52c6a0f.tar.gz#egg=firefox_code_coverage-1.0.0
+firefox_code_coverage
```
Comment (marco-c) [inline requirements.txt:4]: Could you specify an exact version?

Comment (nnrepos) [reply requirements.txt]: done

**Comment by nnrepos** [comment]
it seems like `firefox_code_coverage` has some dependency issue starting from version `1.2.0`:
`ImportError: failed to find libmagic.  Check your installation`
fyi, i think it can be fixed [like this](https://github.com/Yelp/elastalert/issues/1927).
for now, i'm depending on `firefox_code_coverage==1.1.6`.

**Review by marco-c — APPROVED** [review]

**Comment by marco-c** [comment]
Could you remove the `tests/__init__.py` file from the PR, since it's unrelated?

**Review by marco-c — APPROVED** [review]
Thanks!

**Pull request merged**